src/components/file-handler/FileHandler.stories.jsx

```diff
@@ -30,6 +30,8 @@ export default {
   },
 };
 
+const src = 'https://source.unsplash.com/240x240/?cat';
```

Comment:
can we use more static image here? I know it's haw it was defined but maybe it's a good occasion to change it here? 

cc @clxandstuff

Reply:
of course :) We event can use a dog picture 🐶